build.gradle.kts

```diff
@@ -67,6 +67,10 @@ dependencies {
         exclude(module = "opus-java")
     })
 
+    modApi("maven.modrinth:vanish:1.3.2") {
+        isTransitive = false
+    }
```

Comment:
Why not `modImplementation`?

This doesn't need to be forwarded as this isn't exposing vanish as part of its own API.